src/introduction.tex

```diff
@@ -46,15 +46,17 @@
 
 \subsubsection{Statement of the problem}
 
-Nowadays the industry increasingly relies on cloud technologies. More and more companies start or move their products to cloud environment.
-It has been observed that gradually institutions introduce dedicated teams for managing the costs of cloud operations or use third-parties for that purpose. Such solutions cause great savings for everyone.
-Unfortunately existing toolkit is either very primitive or targets billion dollar companies. Currently neither Amazon nor any third-party offer a tool that would allow flexible, cheap and quick insight into the operation costs for small or middle-sized organizations.
+The software industry increasing adopt public cloud as the primary method of running their infrastructure.
+
+As the cost of cloud increases, the companies invest in dedicated teams or tools to improve visibility and drive savings. Most of the current solution aims to reduce the cost of production workloads. However, systems for research and development have very different patterns, poorly addressed by existing tools. 
 
 \subsubsection{The solution}
 
-The Omigost is a system that offers reliable solution adressing that issue. We focus on one of the biggest cloud platforms - Amazon Web Services, which is maintained by Amazon.
-Whole system intends not only to provide easy cost insight, but also help with basic cost optimization.
-Despite collecting and storing AWS data the system will offer users following core features:
+The Omigost is a system that addresses the research and development cost saving use case. We focus on one of the biggest cloud platform - Amazon Web Services, which is managed by Amazon.
```

Comment:
one of the biggest cloud platform**s**

Reply:
Disagree. It is a single platform. The plural would be `the biggest cloud platforms are Azure, AWS, GCP`.

Reply:
yeah, it is a single platform, thus we have the word `one` there - but, it's one *from among a range* of many other *platforms*, that's what the phrase is about, the plural here doesn't indicate we consider a multitude of them, but that the AWS is not the only big cloud provider that is out there

Reply:
some backing:
https://blog.harwardcommunications.com/2012/05/25/one-of-each-of-any-of-plus-plural/